types/stripe/index.d.ts

```diff
@@ -3451,12 +3462,6 @@ declare namespace Stripe {
              */
             charge: string | charges.ICharge | null;
 
-            /**
-             * Whether or not the invoice is still trying to collect payment. An invoice is closed if it's either paid or
-             * it has been marked closed. A closed invoice will no longer attempt to collect payment.
-             */
-            closed: boolean;
```

Comment:
Should these be marked optional and `@deprecated` rather than removed?  Are they still accessible to a Stripe API version people might be using?

Reply:
Thanks for the catch, I was not aware of the `@deprecated`!

I've re-added the items as `@deprecated` and `optional`.